### PR TITLE
Issue #73: adding collection_process_complete

### PR DESCRIPTION
**Pull request by Philly-B**

### CODE REVIEW

**Comment by Philly-B** [comment]
@reckart  Do you mind having a look?

You will see that I also added a new function called `__request_without_response` since I had problems using `__request_with_json_response`. The python client seems to clear the response when the status code is 204, which it is for this call. Therefore I added the method mentioned above which just does not care about the content, however, checks that the status code is indeed 204.

**Comment by reckart** [comment]
Isn't the convention that all calls to the platform should return a JSON response?

**Comment by Philly-B** [comment]
That is indeed the convention, however, the python lib seems to somehow clear the content of the response if the status code is 204.

**Comment by reckart** [comment]
Said differently, collection process complete should probably return a 200 success and not a 204 no content.

**Comment by Philly-B** [comment]
Alright, I changed the backend to only return 200 instead of 204, it will be in the next release. Do you mind finishing the review of this issue?

File: tests/test_pipeline.py [hunk 1]
```diff
@@ -259,3 +259,15 @@ def test_delete_pipeline_v6(client_version_6, requests_mock):
...
+
+
+def test_collection_process_complete(client_version_6, requests_mock):
+    pipeline = Pipeline(Project(client_version_6, "LoadTesting"), "discharge")
+    requests_mock.post(
+        f"{URL_BASE_ID}/rest/experimental/textanalysis/projects/"
+        f"LoadTesting/pipelines/discharge/collectionProcessComplete",
+        headers={"Content-Type": "application/json"},
+        json={"payload": None, "errorMessages": []},
+        status_code=204
```
Comment (reckart) [inline tests/test_pipeline.py:271]: If the backend was changed to return a 200, then this test should also be changed to return a 200.

Comment (Philly-B) [reply tests/test_pipeline.py]: Yes thank you for fixing it!

**Pull request merged**